test/start_test.clj

```diff
@@ -1,6 +1,22 @@
 (ns start-test
   (:require [clojure.test :refer :all]
-            [start]))
+            [start])
+  (:import [org.apache.activemq ActiveMQSslConnectionFactory]))
 
-(deftest adding-numbers
-  (is (= 4 (start/plus 2 2))))
\ No newline at end of file
+(defn with-test-jms-connection
+  "CALL with a local JMS connection for testing."
+  [call]
+  (let [url     "vm://localhost?broker.persistent=false"
+        factory (new ActiveMQSslConnectionFactory url)
+        queue   "test.queue"]
```

Comment:
What is this "vm://localhost?" magic? ;-) Ah. Google explains …

Reply:
Oops, forgot to document this, will add https://activemq.apache.org/how-do-i-embed-a-broker-inside-a-connection here

Reply:
No it's good. I learned something by googling that `vm` URL.